src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -55,7 +59,14 @@ const AnonymousChannelListScreen = ({route}) => {
     if (item?.channelType === 'ANON_TOPIC') {
       // TODO: ADD the correct ANON_TOPIC Channel Item Component here;
 
-      return <MessageChannelItem item={item} onChannelPressed={() => goToChatScreen(item)} />;
+      // return <MessageChannelItem item={item} onChannelPressed={() => goToChatScreen(item)} />;
+      return (
+        <CommunityChannelItem
+          channel={item}
+          onChannelPressed={() => goToCommunityScreen(item)}
+          fetchTopicLatestMessage={fetchLatestTopicPost}
+        />
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `CommunityChannelItem` component is being used for `ANON_TOPIC` channel type. Ensure that this component supports all the necessary features and styles for anonymous topics. If there are significant differences between community channels and anonymous topics, consider creating a separate component for `ANON_TOPIC` to maintain modularity and readability.


<!-- This is an auto-generated comment by OSS CodeRabbit -->